.circleci/config.continue.yml.j2

```
@@ -36,7 +36,7 @@ instrumentation_modules: &instrumentation_modules "dd-java-agent/instrumentation
 debugger_modules: &debugger_modules "dd-java-agent/agent-debugger|dd-java-agent/agent-bootstrap|dd-java-agent/agent-builder|internal-api|communication|dd-trace-core"
 profiling_modules: &profiling_modules "dd-java-agent/agent-profiling"
 
-default_system_tests_commit: &default_system_tests_commit 35bb7f9753dc82b7c8e4a8276bf6f0ccad0e5dc3
+default_system_tests_commit: &default_system_tests_commit 6ece55f02c992f4ecf598472882d5cbe1a77f721
```

```suggestion
default_system_tests_commit: &default_system_tests_commit 315bc8a32cc888834726397f088336ba8038277f
```

There's been a new updated version :smile: